### PR TITLE
no-resized-images-generated

### PR DESCRIPTION
**Pull request by grking8**
Not 100% sure about this, but it solved my issue which resembled closely https://github.com/grantmcconnaughey/django-avatar/issues/153

### CODE REVIEW

**Comment by ls-initiatives** [comment]
The build seems to fail for unrelated reasons.
Can this be merged please ?

**Comment by ntoll** [comment]
Thanks to everyone for their work on django-avatar and this patch too.

This fix doesn't completely work. I used the following PNG file as a test avatar:

python_logo

However, with this patch applied, the resulting resized avatar image looks like this (note the background):

avatar_problems

It appears the solution that works is to set `AVATAR_THUMB_FORMAT = "PNG"` in `settings.py`. Perhaps the default out-of-the-box should be "PNG"..? Thoughts..?

**Comment by ntoll** [comment]
FYI... this is what it looks like with `AVATAR_THUMB_FORMAT = "PNG"`:

avatar_works

**Comment by johanneswilm** [comment]
Hey @grking8 @ntoll @ls-initiatives ,
I am currently looking at what can be merged into django-avatar. It sounds like this sin't quite ready to be merged due to some issues @ntoll found? @ntoll would you be able to provide an updated PR so that we can get this merged?

**Comment by ntoll** [comment]
TL;DR - I don't think this a blocker.

My point isn't so much a bug as a reminder that, on some systems, if you don't specify the `AVATAR_THUMB_FORMAT = "PNG"` you'll get non RGBA (i.e. with a black background, as shown above) resized images if you upload an RGBA avatar image.

**Comment by johanneswilm** [comment]
ok, @ntoll , times are changing and PNG are possibly more suitable in 2022. But then I'm guessing we should stop no longer merge this PR as there is no reason to convert all RGBA files to RGB, is there?

**Comment by johanneswilm** [comment]
Making AVATAR_THUMB_MODES configurable should solve this. Please reopen if the issue has not been resolved entirely.
